tools/dependencies-report/src/main/resources/notices/strscan-NOTICE.txt

```diff
@@ -0,0 +1,56 @@
+StringScanner is copyrighted free software by Minero Aoki.
```

Comment:
This seems little bit different from https://github.com/elastic/logstash/blob/main/tools/dependencies-report/src/main/resources/notices/strscan-NOTICE.txt